src/main/java/org/stellar/sdk/StrKey.java

```diff
@@ -23,7 +25,8 @@ class StrKey {
 
   public static final int ACCOUNT_ID_ADDRESS_LENGTH = 56;
   private static final byte[] b32Table = decodingTable();
-  private static final Base32 base32Codec = new Base32();
+  private static final Base32 base32Codec =
```

Comment:
this doesn't seem to be necessary to configure with additional settings and STRICT, since `base32decode()` determines validity now, and it may lead to partial duplication of functionality. I tried reverting this line locally, and the tests still pass.